libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.java

```diff
@@ -70,11 +67,13 @@ public static Builder builder(Context context) {
   }
 
   static Builder builder(Context context, LocaleUtils localeUtils) {
+    AccountsManagerImpl accountsManager = AccountsManagerImpl.getInstance(context);
```

Comment:
why are we accessing impl vs. interfaces here? also what's the plan with the testing of this logic? should we be injecting `AccountManager` instead?